test/e2e/framework.go

```diff
@@ -386,3 +387,20 @@ func kubectlExec(namespace string, podName, containerName string, args ...string
 	err := cmd.Run()
 	return stdout.Bytes(), stderr.Bytes(), err
 }
+
+// Wrapper to ginkgo functions for e2e tests which can be easily filtered via focus tags.
+func KubeIt(text string, body interface{}, timeout ...float64) bool {
+	tname := "["+KubeE2ETestTag+"] "+text;
+	if len(timeout) == 0 {
```

Comment:
Timeouts in Ginkgo are fundamentally broken and we shouldn't use them. See #8672 and onsi/ginkgo#165.